src/aura/CMP_Acc_Naming_Dropdown/CMP_Acc_Naming_DropdownHelper.js

```diff
@@ -0,0 +1,94 @@
+({
+    init : function(component) {
+        //Retrieving admin account naming options
+        this.getAdminAccNameFormatOptions(component);
+        this.getHHAccNameFormatOptions(component);
+    },
+    onSelectChange : function(component) {
+        var selectedVal = component.find("nameFormatDropDown").get("v.value");
+        var prefix = component.get("v.namespacePrefix");
+        if (prefix === 'hed__')
+        {
+            if (selectedVal === ($A.get("$Label.hed.acctNamingOther")))
+            {
+                component.set("v.otherDisplay", true);
+            }else{
+                component.set("v.otherDisplay", false);
+            }
+        }else{
+            if (selectedVal === ($A.get("$Label.c.acctNamingOther")))
+            {
+                component.set("v.otherDisplay", true);
+            }else{
+                component.set("v.otherDisplay", false);
+            }
+        }
+        component.set("v.nameFormat", selectedVal);
+    },
+    
+    saveSetting : function(component) {        
+        //Get selected value
+        var selectedVal = component.find("nameFormatDropDown").get("v.value");
+        component.set("v.setting", selectedVal);
+
+        //Set the selected value in the settings (so it gets saved)
+        var prefix = component.get("v.namespacePrefix");
+        if (prefix === 'hed__')
+        {
+            if (selectedVal === ($A.get("$Label.hed.acctNamingOther")))
+            {
+                selectedVal = component.find("otherText").get("v.value");
+                component.set("v.otherSetting", selectedVal);
+            }
+        }else{
+            if (selectedVal === ($A.get("$Label.c.acctNamingOther")))
+            {
+                selectedVal = component.find("otherText").get("v.value");
+                component.set("v.otherSetting", selectedVal);
+            }
+        }
+    },
+
+    getAdminAccNameFormatOptions : function(component) {
+        var prefix = component.get("v.namespacePrefix");
+        var adminAccNameFormatOptions = [];
+        if (prefix === 'hed__')
+        {
+            adminAccNameFormatOptions.push($A.get("$Label.hed.lastNameAdminAcc"));
```

Comment:
Looks like the options here is different from the requirement. 

For example: 
In the requirement, it should be:  {!LastName} ({!{!FirstName}}) Administrative Account: Generates an Administrative Account name such as “Smith (John) Administrative Account”.

But the option is: {!LastName} {!FirstName} Administrative Account: And it generates Smith John Administrative Account.  The '(' and ')' is missing.

Reply:
Good catch! I will fix this.